client/src/utils/useForm.js

```diff
@@ -0,0 +1,59 @@
+import { useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+import { customerOrders } from '../actions/customer';
+import { removeOrdered } from '../reducers/cartReducer';
+import validateInfo from './validateInfo';
+
+const useForm = () => {
```

Comment:
This hook describes `credit card form` handling only, so use more specific name for this hook.